modules/sublimeBidAdapter.js

```diff
@@ -155,25 +160,25 @@ export const spec = {
       };
 
       if (!response.cpm) {
-        sendAntennaPixel('dirnocpm');
+        sendAntennaPixel('dirnocpm', requestId);
```

Comment:
requestId is declared in buildRequests method : maybe we have to use bidResponse.requestId here